patches/chrome-browser-ui-webui-cr_components-theme_color_picker-theme_color_picker_handler.h.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/ui/webui/cr_components/theme_color_picker/theme_color_picker_handler.h b/chrome/browser/ui/webui/cr_components/theme_color_picker/theme_color_picker_handler.h
+index d3dafeaa60c3622106ef5f17dc0688e2f16ce4bd..b21ee9332e81b02294b24b055e9a596837a8bff5 100644
+--- a/chrome/browser/ui/webui/cr_components/theme_color_picker/theme_color_picker_handler.h
++++ b/chrome/browser/ui/webui/cr_components/theme_color_picker/theme_color_picker_handler.h
+@@ -59,6 +59,7 @@ class ThemeColorPickerHandler
+   void UpdateTheme() override;
+ 
+  private:
++  friend class BraveThemeColorPickerHandler;
```

Comment:
this patch does not follow patching standards, but I'm a bit concerned about this change in general. Is this a temporary workaround that is fixed in a later commit?

Reply:
I'll fix the path, but it's essentially reverting us to a two-color picker by reinstating the upstream code that was calling `CreateChromeColor` when `!IsChromeRefresh2023()` was true.

Reply:
https://github.com/brave/brave-browser/issues/39045 cc @rebron